### PR TITLE
L16 dev config without genesis deposits pre-allocation

### PR DESCRIPTION
**Pull request by atif-konasl**
L16 dev config without genesis deposits pre-allocation

### CODE REVIEW

**Review by Leondroids — DISMISSED** [review]
LGTM

File: l16/dev/config.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Extends the mainnet preset
+PRESET_BASE: 'mainnet'
+# Genesis
+# ---------------------------------------------------------------
+MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: 1
```
Comment (mxmar) [inline l16/dev/config.yaml:5]: We should decide on some value. If we have 100 genesis validators, then let's use `100` here.

File: l16/dev/config.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+# Fri Jan 14 2022 15:57:24 GMT+0100 (Central European Standard Time)
+MIN_GENESIS_TIME: 1642172244
+# Gensis fork
+GENESIS_FORK_VERSION: 0x62000069
+# 300 seconds (5 min)
+GENESIS_DELAY: 120
```
Comment (mxmar) [inline l16/dev/config.yaml:11]: IMHO let's do delay with default `300`s

File: l16/dev/genesis.json [hunk 3]
```diff
@@ -0,0 +1,862 @@
...
+      "storage": {
+        "0x00000000000000000000000000000000000000000000000000000000000000":"0x7880bf33442b07d1af22347c74f6edfc0a155fb03be9f0348a553c75dac70184",
```
Comment (mxmar) [inline l16/dev/genesis.json:790]: In our case shouldn't we start network with `0x00000000000000000000000000000000000000000000000000000000000020` (0 deposit txs?)

File: l16/dev/l16_runner.sh [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+    nohup ./bin/geth  \
+        --catalyst \
+        --datadir $BASE_PATH/kintsugi/datadir/geth \
+        --networkid 1337222 \
+        --http \
+        --http.api "engine,net,eth,admin,debug,miner" \
+        --http.addr 0.0.0.0 \
+		--ws \
+        --ws.api "engine,net,eth,admin,debug,miner" \
+        --ethstats atif-l15-test@dev.stats.pandora.l15.lukso.network \
```
Comment (mxmar) [inline l16/dev/l16_runner.sh:18]: `atif-l15-test` hardcoded value, maybe we should use script argument to provide name? (like `$1`)

File: l16/dev/l16_runner.sh [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+	sudo kill -INT $(sudo lsof -t -i:30303) &>/dev/null
+	sleep 1
+
+	echo "shutting down eth2 stats client..."
+	sudo pkill "eth2stats-client*"
+    sleep 1
+
+    echo "cleanning up datadir..."
+    rm -rf $BASE_PATH/kintsugi/datadir/geth/geth;
+    rm -rf $BASE_PATH/kintsugi/datadir/beacon/beaconchaindata $BASE_PATH/kintsugi/datadir/beacon/metaData;
```
Comment (mxmar) [inline l16/dev/l16_runner.sh:126]: You should also delete `tosaccepted` file here: `$BASE_PATH/kintsugi/datadir/beacon/tosaccepted`

File: l16/dev/l16_runner.sh [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+    rm -rf $BASE_PATH/kintsugi/datadir/validator/validator.db;
```
Comment (mxmar) [inline l16/dev/l16_runner.sh:127]: You should also delete `tosaccepted` file here: `$BASE_PATH/kintsugi/datadir/validator/tosaccepted`

**Review by mxmar — CHANGES_REQUESTED** [review]
I left comments. We should think about how to customize this script. How to provide availability to provide for example own address: `miner.etherbase=0xcce25d1620bd5e33485b808b30d3c805ea28dbe`

File: l16/dev/config.yaml (hunk 1 above)
Comment (atif-konasl) [reply l16/dev/config.yaml:5]: > I left comments. We should think about how to customize this script. How to provide availability to provide for example own address: `miner.etherbase=0xcce25d1620bd5e33485b808b30d3c805ea28dbe`

I think the script could help us with command that's why i added that script.

Comment (atif-konasl) [reply l16/dev/config.yaml:5]: > We should decide on some value. If we have 100 genesis validators, then let's use `100` here.

Thanks. I am changing it.

File: l16/dev/genesis.json [hunk 7]
```diff
@@ -0,0 +1,862 @@
+{
+  "config": {
+    "chainId": 1337222,
```
[the remark this answers is not in the document]
Comment (atif-konasl) [reply l16/dev/genesis.json:3]: Actually i have added my test's configs.

**Pull request closed without merging**